HW_3/task_2.py

```diff
@@ -0,0 +1,30 @@
+"""None."""
+# В большой текстовой строке подсчитать количество встречаемых слов и вернуть 10 самых
+# частых повторяющихся. Не учитывать знаки препинания и регистр символов. За основу возьмите любую
+# статью из википедии или из документации к языку.
+import collections
+
+TEXT = 'Вышедшая в свет в 1959 году книга «Электронные цифровые машины и программирование»' \
+       ' (Китова А. И. и Криницкого Н. А.) была первым в СССР официальным учебником по ЭВМ и ' \
+       'программированию, официально допущенным Министерством образования СССР для обучения в ' \
+       'ВУЗах и университетах. Об этой книге Президент Академии наук СССР Г. И. Марчук писал: ' \
+       '«В 1959 году появился другой фундаментальный труд А. И. Китова, написанный вместе с Н. А. ' \
+       'Криницким — „Электронные цифровые машины и программирование“. Это была фактически ' \
+       'энциклопедия науки об ЭВМ. Многие поколения студентов в университетах и вузах страны с ' \
+       'помощью этой замечательной книги получили фундаментальное образование и стали ' \
+       'первоклассными учёными во многих областях знаний. Книги А. И. Китова, написанные ' \
+       'в начале эры ЭВМ в нашей стране, не должны быть забыты». Эта книга была издана в Румынии, ' \
+       'Венгрии, Германской демократической республике и ряде других стран. Второе стереотипное ' \
+       'издание книги «Электронные цифровые машины и программирование» появилось в 1961 году. ' \
+       'Общий тираж иностранных и двух советских изданий составил свыше 130 000 экземпляров. ' \
+       'Об этом учебнике-энциклопедии в своих воспоминаниях ветеран кафедры вычислительной техники ' \
+       'МЭИ (первая кафедра ЭВМ в стране), д. т. н., профессор А. К. Поляков написал так: «По-моему ' \
+       'мнению, учебник А. И. Китова и Н. А. Криницкого „Электронные цифровые машины и ' \
+       'программирование“ (1959) в то время был лучшим в мире»'
+WORD_LIST = 10
+punctuation_set = {".", ";", ":", "!", "?", "/", "\\", ",", "#", "@", "$", "&", ")", "(", "'", "\""}
+new_list = ''.join(value if value not in punctuation_set else ' ' for value in TEXT).split()
```

Comment:
Не обязательно не обязательно пунктуацию заменять на пробел, можно просто ее исключить.
```suggestion
new_list = ''.join(value for value in TEXT if value not in punctuation_set).split()
```